asdf_astropy/converters/fits/fitswcs.py

```diff
@@ -0,0 +1,30 @@
+from asdf.extension import Converter
+
+
+class FitsWCSConverter(Converter):
+    """
+    Converter for serializing and deserializing `astropy.wcs.WCS` objects.
+
+    This converter currently supports the serialization of simple WCS objects
+    by preserving the `wcs.to_header()` data. It does not support complex WCS objects
+    such as tabular or distortion WCSes.
+
+    Future work:
+        - Until the support for tabular and distortion WCSes is added,
+          throw error for such WCSes when passed through in the converter
+        - Implement mechanisms to detect tabular and distortion WCSes and support their serialization
+    """
+
+    tags = ("tag:astropy.org:astropy/fits/fitswcs-*",)
+    types = ("astropy.wcs.wcs.WCS",)
+
+    def from_yaml_tree(self, node, tag, ctx):
+        from astropy.wcs import WCS
+
+        header = node["header"]
+        return WCS(header)
+
+    def to_yaml_tree(self, wcs, tag, ctx):
+        node = {}
+        node["header"] = dict(wcs.to_header())
```

Comment:
This loses the header comments. I'm not opposed to that (for now) but is there an easy way to preserve these?

Reply:
Does the WCS object store the comments at all? I didn't think it did.

Reply:
Yes the comments are preserved. Reading an `astropy.wcs.WCS` from a random JWST file and calling `wcs.to_header` I see the comments.
```
WCSAXES =                    2 / Number of coordinate axes
CRPIX1  =                160.5 / Pixel coordinate of reference point
CRPIX2  =                160.5 / Pixel coordinate of reference point
...
```